features/mbedtls/targets/TARGET_STM/TARGET_STM32F4/TARGET_NUCLEO_F439ZI/mbedtls_device.h

```diff
@@ -21,6 +21,7 @@
 #define MBEDTLS_DEVICE_H
 
 #define MBEDTLS_AES_ALT
+#define MBEDTLS_SHA1_ALT
 
-
+#define MBEDTLS_SHA1_C
```

Comment:
Hi @LMESTM ,
This define is supposed to be located in features\mbedtls\inc\mbedtls\config.h.
It would enable the SHA1 module for every targets, thus adding code size for everyone.

I have chosen to add the SHA1 enablement in this target only.